### DISCUSSION TITLE
Event modifier: "immediate"

### DISCUSSION THREAD

**Discussion by pacmanoncrack** [alpinejs/alpine#2636] [3. Feature ideas]
I think it would be a really good idea to include the event-modifier `Event.stopImmediatePropagation()` in the same way as `.prevent` or `.stop`. There are a lot of use-cases and I find myself writing the method quite often in projects.

**Comment by dy** [comment]
[Sprae](https://github.com/dy/sprae) supports .stop.immediate modifier if you want to have a look.
